src/cairo_run.rs

```diff
@@ -24,6 +24,7 @@ pub struct CairoRunConfig<'a> {
     pub print_output: bool,
     pub layout: &'a str,
     pub proof_mode: bool,
+    pub secure_run: Option<bool>,
```

Comment:
```suggestion
    pub secure_run: bool,
```

Reply:
This is the original format. 
The logic for determining its value is
if Some(bool) -> bool
if None -> if proof_mode: false else true
This logic cant be applied with a simple boolean value